x-pack/legacy/plugins/maps/public/connected_components/map/feature_tooltip.js

```diff
@@ -228,6 +254,28 @@ export class FeatureTooltip extends React.Component {
     });
   }
 
+  _filterGeoFields(feature) {
+    if (!feature) {
+      return [];
+    }
+
+    // line geometry can only create filters for geo_shape fields.
+    if (feature.geometry.type === GEO_JSON_TYPE.LINE_STRING
+      || feature.geometry.type === GEO_JSON_TYPE.MULTI_LINE_STRING) {
+      return this.props.geoFields.filter(({ geoFieldType }) => {
+        return geoFieldType === ES_GEO_FIELD_TYPE.GEO_SHAPE;
+      });
+    }
+
+    // TODO support geo distance filters for points
+    if (feature.geometry.type === GEO_JSON_TYPE.POINT
```

Comment:
why not `intersects` on points? e.g. show me the state that this point is in.

Create github issue for distance-filter support instead of todo. Also, would that even apply here? Wouldn't a filter-by-distance be a separate action, in addition to shape filtering?

Reply:
geo_points field type does not support [geo shape query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-geo-shape-query.html) so its not possible to do a intersects query with geo_point fields. geo_point fields only support geo_bounding_box, geo_distance, and geo_polygon

Reply:
discussed this in person. this would create a weird asymmetry, where filters from points would have to omit geo_point-layers. omitting filtering altogether then.